MykSuite/src/main/java/com/infomaniak/lib/myksuite/ui/components/MyKSuiteChips.kt

```diff
@@ -18,62 +18,48 @@
 package com.infomaniak.lib.myksuite.ui.components
 
 import android.content.res.Configuration
+import androidx.annotation.DrawableRes
 import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
 import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.shape.RoundedCornerShape
-import androidx.compose.material3.SuggestionChip
-import androidx.compose.material3.SuggestionChipDefaults
 import androidx.compose.material3.Surface
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.res.vectorResource
 import androidx.compose.ui.tooling.preview.Preview
 import com.infomaniak.lib.myksuite.R
 import com.infomaniak.lib.myksuite.ui.theme.Dimens
+import com.infomaniak.lib.myksuite.ui.theme.Margin
 import com.infomaniak.lib.myksuite.ui.theme.MyKSuiteTheme
 
 @Composable
-fun MyKSuitePlusChip(modifier: Modifier = Modifier, onClick: () -> Unit = {}) {
-    ThemedChip(
-        modifier = modifier,
-        label = { Image(ImageVector.vectorResource(R.drawable.ic_logo_my_ksuite_plus), contentDescription = "My kSuite +") },
-        onClick = onClick,
+fun MyKSuiteChip(modifier: Modifier = Modifier, @DrawableRes imageRes: Int) {
```

Comment:
Instead of bothering with passing imageRes as arguments, there could be a parameter such as a boolean or an enum that directly tells the chip which type to display as. But this will be done in a later PR